### ISSUE TITLE
arabic subtitle issue

### ISSUE THREAD

**Issue by sigmazero123** [Coi-l/PodnapisiSubtitles.bundle#7]
when i choose any subtitle agent like (odnapisi Subtitles,..) to download arabic subtitle and english subtitle for ever what i watch the arabic subtitle downloaded form this agents doesn't displayed correctly ! although the english srt is shown perfect

![](https://15254b2dcaab7f5478ab-24461f391e20b7336331d5789078af53.ssl.cf1.rackcdn.com/plex.vanillacommunity.com/editor/xv/2z3xoxnk66bq.jpg)

PMS :0.9.12.11.1406-8403350
Samsung UA46F8000 (2013) model
Plex version 2.003

**Comment by Coi-l** [comment]
This might be a silly question, but does any arabic subtitles display correctly in Plex?

This is probably a problem with the format of the text (ascii, utf8 etc) but I want to make sure that Plex actually can display arabic correctly before I dig into this.

**Comment by sigmazero123** [comment]
yes the newest version from orca 2.003 fixed the arabic subtitle issue but sadly the local subtitle only 


![](https://15254b2dcaab7f5478ab-24461f391e20b7336331d5789078af53.ssl.cf1.rackcdn.com/plex.vanillacommunity.com/editor/49/kht2nq1uv3yw.jpg)

you can read more from this topic 
http://forums.plex.tv/discussion/comment/1025505#Comment_1025505

**Comment by Coi-l** [comment]
Hey, just wanted to drop a line saying I haven't forgotten about this.

Currently however I have a really limited amount of time free during my off-work hours (3 months old kid at home), so I don't know when I'll be having the time to dig into this. 

I'll do my best.

Cheers

**Comment by sigmazero123** [comment]
sure thing ..am here for you if you want any help ..for testing ..thanks dude ;)
